doc/src/Intro.txt

```diff
@@ -0,0 +1,43 @@
+"Previous Section"_Manual.html - "LAMMPS WWW Site"_lws -
+"LAMMPS Documentation"_ld - "LAMMPS Commands"_lc - "Next
+Section"_Section_start.html :c
+
+:link(lws,http://lammps.sandia.gov)
+:link(ld,Manual.html)
+:link(lc,Commands.html#comm)
+
+:line
+
+Introduction :h2
+
+The "LAMMPS website"_lws is the best introduction to LAMMPS.
+
+Here is a list of webpages you can browse:
```

Comment:
This may be one of the first pages a newcomer might see and can be very confusing. Where should you start? What is important? Normally you would want to make the documentation in a way so it is easier for people to get started. But this is the reverse. So, I'm completely against this change.